src/binaryen-c.cpp

```diff
@@ -163,6 +163,32 @@ BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const
   return ret;
 }
 
+BinaryenFunctionTypeRef BinaryenGetFunctionType(BinaryenModuleRef module, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams) {
+  auto* wasm = (Module*)module;
+  auto* ret = new FunctionType;
```

Comment:
Just copied this from the respective add function to make sure that I don't accidentally break something.